scripts/tests/config/test_apm_managed.yml

```diff
@@ -0,0 +1,34 @@
+apm-server:
+  container_name: localtesting_8.0.0_apm-managed
+  depends_on:
+    kibana:
+      condition: service_healthy
+  environment:
+    FLEET_ELASTICSEARCH_HOST: http://elasticsearch:9200
+    FLEET_SERVER_ELASTICSEARCH_INSECURE: "1"
+    FLEET_SERVER_ENABLE: "1"
+    FLEET_SERVER_HOST: 0.0.0.0
+    FLEET_SERVER_INSECURE_HTTP: "1"
+    FLEET_SERVER_POLICY_ID: fleet-server-apm-policy
+    FLEET_SERVER_PORT: "8220"
+    FLEET_SERVER_SERVICE_TOKEN: AAEAAWVsYXN0aWMvZmxlZXQtc2VydmVyL2VsYXN0aWMtcGFja2FnZS1mbGVldC1zZXJ2ZXItdG9rZW46bmgtcFhoQzRRQ2FXbms2U0JySGlWQQ
```

Comment:
Are these service tokens for the connection to Elasticsearch, as documented here? https://www.elastic.co/guide/en/elasticsearch/reference/current/service-tokens-command.html

Reply:
yep, they are those tokens 
https://www.elastic.co/guide/en/fleet/current/add-a-fleet-server.html
```
Notes:

Make sure you download an x64 architecture installation package.
For the agent policy, choose one with a Fleet Server integration, or click Create policy to create one now. Alternatively you can [create a Fleet Server policy without using the UI](https://www.elastic.co/guide/en/fleet/8.1/create-a-policy-no-ui.html), then select the policy here.
If you choose Production deployment mode, learn how to generate certs in [Encrypt traffic in clusters with a self-managed Fleet Server](https://www.elastic.co/guide/en/fleet/8.1/secure-connections.html).
It’s recommended you generate a unique service token for each Fleet Server. For other ways to generate service tokens, see [elasticsearch-service-tokens](https://www.elastic.co/guide/en/elasticsearch/reference/8.1/service-tokens-command.html).
```